lib/keyResolver.js

```diff
@@ -1,9 +1,15 @@
+const didIo = require('did-io');
+// Config did-io to support did:key driver
+didIo.use('key', require('did-method-key').driver());
```

Comment:
Need to use `import` not `require`.

Reply:
I will fix this in a second @dmitrizagidulin I think we are good here.

Reply:
Addressed here: https://github.com/digitalbazaar/bedrock-tokenization/pull/2/commits/0ea252be34d3dd3f626b11e7a8e9c9a3cafcfca7